staking_deposit/settings.py

```diff
@@ -1,7 +1,7 @@
 from typing import Dict, NamedTuple
 
 
-DEPOSIT_CLI_VERSION = '1.2.0'
+DEPOSIT_CLI_VERSION = '2.0.0'
```

Comment:
[not strongly] nitpicking it's better to be in another PR.

Reply:
👍 agreed it was a lazy move. I have reverted it in https://github.com/ethereum/eth2.0-deposit-cli/pull/240/commits/e562b5f8cd19513460d39d18fc86213a6164036d

will open new PR to adress